### PR TITLE
Simplify member list of access groups

### PR DESCRIPTION
**Pull request by jens1o**

### CODE REVIEW

**Comment by jens1o** [comment]
@fiji-flo Could you please take a look overall? From what I see I'm "done" with the refactoring of the access groups. Still, I'd like to allocate some time tomorrow for more in-deep testing.

File: src/store/access-group.store.js [hunk 1]
```diff
@@ -155,18 +155,49 @@ export default {
...
+      return invitationConfig ? invitationConfig.content : null;
+    },
+    isCurator: ({ isCurator }) => {
+      console.log('access-group.store.js#isCurator was called');
+      return isCurator;
+    },
+    isMember: ({ isMember }) => {
+      console.log('access-group.store.js#isMember was called');
+      return isMember;
+    },
```
Comment (jens1o) [inline src/store/access-group.store.js:201]: I'm going to check tomorrow whether any other implementation in the Frontend relies on these methods and if not, I'd opt to remove them (and the hole file)

File: src/components/access_group/AccessGroupEdit.vue [hunk 2]
```diff
@@ -135,8 +124,65 @@ export default {
...
+        const memberData = await accessGroupApi.execute({
+          path: 'members/get',
+          endpointArguments: [this.groupname, { sort: 'role-asc' }],
+        });
+        const members = memberData.members.map(
+          (member) => new DisplayMemberViewModel(member),
+        );
+        this.memberList = members;
+      } catch (e) {
+        console.log(e.message);
```
Comment (fiji-flo) [inline src/components/access_group/AccessGroupEdit.vue:182]: Looks like this try catch block is everywhere. Moving the `console.log` to the `execute` method allows us to remove those (and maybe use `console.error`)

**Review by fiji-flo — COMMENTED** [review]
Looking really good.

File: src/components/access_group/AccessGroupTerms.vue [hunk 3]
```diff
@@ -22,7 +22,7 @@
       <h1 class="group-terms__header">
         {{ fluent('access-group_terms', 'heading') }}
       </h1>
-      <p>{{ termsContent }}</p>
+      <p v-html="formattedTerms"></p>
```
Comment (jens1o) [inline src/components/access_group/AccessGroupTerms.vue:25]: The original implementation seemed to lack actually presenting the TOS as markdown?

**Comment by jens1o** [comment]
Status update: If I'm not mistaken, I've reviewed all Access Group compoentents to not include any store reference (with the exception of loading mechanisms). I'll now tackle the synchronization after (partial) data updates.

**Review by fiji-flo — APPROVED** [review]

**Comment by jens1o** [comment]
closing in favor of #626 (having all commits signed)

**Pull request closed without merging**